### DISCUSSION TITLE
JavaScript

### DISCUSSION THREAD

**Discussion by rebazjabar** [vercel/next.js#44242] [Help]
hello dears
this project is TypeScript im not huge fan of TypeScript ,maybe there is more 
can you make the project the same as this but with JavaScript language ?
or how to change this project Language to javaScript that work properly ?

**Comment by nordowl** [comment]
When you create a new Next.js project using `npx create-next-app@latest`, you get asked wether you want to use TypeScript or not. TypeScript is optional and you can create a Next.js app without it.
